tests/btc/transactions.test.ts

```diff
@@ -161,58 +162,45 @@ describe('bitcoin transactions', () => {
     const unspent3Value = 250000;
     const totalUnspentValue = unspent1Value + unspent2Value + unspent3Value;
 
-    const utxos: Array<UTXO> = [
+    const utxos: Array<UTXO & { address: string; blockHeight?: number }> = [
```

Comment:
This test seems like it was changed quite a bit and there are variables in it that are no longer used; it also doesn't seem to be testing the function as thoroughly as it used to 🤔 Should it not still have worked with a slight tweak to take into account the different UTXO order? I think we should put it back to what it was and try figure out if it legitimately broke and we need to fix the logic or try tweak the test to fix it for the new logic.